flink/v1.17/flink/src/test/java/org/apache/iceberg/flink/source/TestFlinkSourceConfig.java

```diff
@@ -46,8 +47,11 @@ public void testFlinkHintConfig() {
     assertThat(result).hasSize(3);
   }
 
-  @Test
+  @TestTemplate
   public void testReadOptionHierarchy() {
+    // TODO: FLIP-27 source doesn't implement limit pushdown yet
+    Assumptions.assumeFalse(useFlip27Source);
```

Comment:
```suggestion
    assumeThat(useFlip27Source).isFalse();
```
we've switched to AssertJ assumptions

Reply:
fixed